Win11ThemeTest/ComboBoxTest.cs

```diff
@@ -0,0 +1,130 @@
+            Mouse.Click(cmbBoxButton.GetClickablePoint());
+            Wait.UntilInputIsProcessed(TimeSpan.FromMilliseconds(500));
+            comboWindow = MainWindow.FindFirstDescendant(cf => cf.ByName("ComboBoxWindow")).AsWindow();
+
+            comboBox = comboWindow.FindFirstDescendant(cf => cf.ByAutomationId("comboBoxList")).AsComboBox();
+            comboBoxBind = comboWindow.FindFirstDescendant(cf => cf.ByAutomationId("comboBoxBind")).AsComboBox();
+            comboBoxBind2 = comboWindow.FindFirstDescendant(cf => cf.ByAutomationId("comboBoxBind2")).AsComboBox();
+        }
+
+        [Test]
+        public void cb_findComboBox()
+        {
+            Assert.IsNotNull(comboWindow);            
+            Assert.IsNotNull(comboBox);
+            Assert.IsNotNull(comboBoxBind);
+            Assert.IsNotNull(comboBoxBind2);
+        }
+
+        [Test]
+        public void cb1_defaultSelectedItem()
+        {
+            Assert.That(comboBox.SelectedItem, Is.Not.Null);           
+        }
+
+        [Test]
+        public void cb1_isNull()
+        {
+            Assert.That(comboBox, Is.Not.Null);           
+        }
+
+        [Test]
+        public void cb1_readEditable()
+        {
+            Assert.That(comboBox.IsReadOnly, Is.False);
+            Assert.That(comboBox.IsEditable, Is.False);            
+        }
+
+        [Test]
+        public void cb2_select()
+        {
+            comboBox.Select("Red");
+            Assert.That(comboBox.SelectedItem.Name, Is.EqualTo("Red"));            
+        }
+
+        //Check if the drop-down is open by clicking on both the drop-down & the drop-down Arrow.
+        [Test]       
+        public void cb2_comboExpandCollapse()
+        {                   
+            comboBox.Expand();
+            Assert.That(comboBox.ExpandCollapseState, Is.EqualTo(ExpandCollapseState.Expanded));
+            comboBox.Collapse();
+            Assert.That(comboBox.ExpandCollapseState, Is.EqualTo(ExpandCollapseState.Collapsed));            
+        }
+        [Test]
+        public void cb2_IsEditable()
+        {           
+            Assert.That(comboBoxBind.IsEditable, Is.True);            
+        }
+
+        [Test]
+        public void cb3_editableTextTest()
+        {          
+            Assert.That(comboBoxBind, Is.Not.Null);
+            Assert.That(comboBoxBind.IsEditable, Is.True);
+            comboBoxBind.EditableText = "10";
+            Assert.That(comboBoxBind.SelectedItem, Is.Not.Null);
+            Assert.That(comboBoxBind.SelectedItem.Text, Is.EqualTo("10"));
+        }
+
+        //Check whether the dropdown is clickable or not.
+        [Test]
+        public void cb4_mouseClick()
+        {
+            Mouse.MoveTo(comboBox.GetClickablePoint());
+            Mouse.Click();
+            Assert.That(comboBox.ExpandCollapseState, Is.EqualTo(ExpandCollapseState.Expanded));         
+          
+        }
+        [Test]
+        public void cb4_mouseSelectClick()
```

Comment:
check this test once, it is failing for me with exception - Message: 
  Expected string length 5 but was 4. Strings differ at index 0.
  Expected: "Green"
  But was:  "Blue"